client/components/thank-you-v2/footer/index.tsx

```diff
@@ -54,7 +55,11 @@ type ThankYouFooterProps = {
 
 export default function ThankYouFooter( { details }: ThankYouFooterProps ) {
 	return (
-		<div className="thank-you__footer">
+		<div
+			className={ classNames( 'thank-you__footer', {
+				'has-multiple-details': details && details.length > 1,
```

Comment:
This allows us to port over this feature https://github.com/Automattic/wp-calypso/pull/85478 for rendering footer details in full width when there's only one single blurb to be rendered.